lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -458,10 +459,10 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
               return false;
           return true;
         };
-        if (!check(aScale) || !check(aZp) || !check(bScale) || !check(bZp) ||
-            !check(cScale) || !check(cScale))
+        if (!check(aScale) || !check(aZp) || !check(cScale) || !check(cScale))
```

Comment:
Is it `cZp` for the last one?

Reply:
Yeah, it should be `cZp`. Thanks for the catch. I have made the change.